internal/generator/templates/packing_context_decode.go.tmpl

```diff
@@ -29,7 +29,7 @@
         if {{ goExpression $scope $field.OptionalClause }} {
     {{- end }}
     {{- if not (eq $field.Alignment 0) }}
-        ztype.AlignReader(r, {{ $field.Alignment }})
+        r.Align({{ $field.Alignment }})
```

Comment:
This needs to handle an error returned by `Align()`

Reply:
I agree, at the same time I am not changing any behaviour (previously we were just silently ignoring the error). Is it OK if I address this in a separate PR?